src/Controller/Report/OpenIssuesController.php

```diff
@@ -35,7 +35,7 @@ protected function configure()
         $post = $request->request;
         $get = $request->query;
 
-        $this->cutoff_days = $get->get('cutoff_days');
+        $this->cutoff_days = (int) $get->get('cutoff_days');
```

Comment:
instead of casting, there's actually `getInt` method, and fallback to `null`.

i think `null` is more valid approach for `not defined` behavior.

Reply:
Yeah, getInt() perfectly fits here. Will use it.